tests_metricflow/integration/query_output/test_query_output.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+import pytest
+from _pytest.fixtures import FixtureRequest
+from metricflow_semantics.specs.query_param_implementations import MetricParameter
+from metricflow_semantics.test_helpers.config_helpers import MetricFlowTestConfiguration
+
+from metricflow.engine.metricflow_engine import MetricFlowQueryRequest
+from metricflow.protocols.sql_client import SqlClient
+from tests_metricflow.integration.conftest import IntegrationTestHelpers
+from tests_metricflow.snapshot_utils import assert_str_snapshot_equal
+
+
+@pytest.mark.sql_engine_snapshot
```

Comment:
Since aliases are consistent across SQL-engines, you can add `@pytest.mark.duckdb_only` to reduce the number of snapshots. Also applies to other tests.